boxoffice/views/participant.py

```diff
@@ -64,11 +67,22 @@ def assign(order):
             line_item.current_assignee.details = assignee_details
             db.session.commit()
         else:
+            if not line_item.is_transferable and line_item.current_assignee is not None:
```

Comment:
Actually, we want this:
```python
if line_item.current_assignee is None or line_item.is_transferable:
    # Allow
else:
    # Deny
```
Since there is no Allow block, we can get to the Deny block by applying a `not`:
```python
if not (line_item.current_assignee is None or line_item.is_transferable):
```

Reply:
Previous incident of clarity-based if/else constructs: https://github.com/hasgeek/coaster/blob/5952d8eb7ed51b8405ace6d82ab71843abe971e2/coaster/sqlalchemy/immutable_annotation.py#L59-L66

There's one other example somewhere I'm unable to find. We had an elaborate if condition broken down into multiple if/pass statements with a final else.